### PR TITLE
feat: Port sample to WinUI, fix broken LVI color, add feature

### PR DESCRIPTION
**Pull request by lukeblevins**
This PR ports the sample to WinUI 3 as per @sasakrsmanovic's request.

ListViewItems should also now alternate their color without hover.
image

### CODE REVIEW

**Comment by sasakrsmanovic** [comment]
Thanks Luke!

**Comment by jeromelaban** [comment]
CI will be fixed by https://github.com/unoplatform/Uno.Samples/pull/86

File: UI/XamlBasics/ListViewSample/ListViewSample.Mobile/Android/AndroidManifest.xml [hunk 1]
```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (nickrandolph) [inline UI/XamlBasics/ListViewSample/ListViewSample.Mobile/Android/AndroidManifest.xml:1]: @lukeblevins why has the mobile head been removed from this sample?

**Comment by nickrandolph** [comment]
@lukeblevins can you rebase and fix any issues with the CI please

**Comment by nickrandolph** [comment]
@lukeblevins can you assist with rebasing this PR so that we can merge it?

**Pull request closed without merging**